frontend/tests/e2e/signalr-tests.ts

```diff
@@ -0,0 +1,39 @@
+import { Overrides }   from "puppeteer";
+import PuppeteerHelper from "../puppeteer-helper";
+//-----------------------------------------------------------------------------
+const conditionalDescribe = process.env.LOCAL_DEV
+    ? describe
+    : describe.skip;
+//-----------------------------------------------------------------------------
+conditionalDescribe("SignalR", () => {
+    const baseUrl = "http://localhost:8080";
+    //-------------------------------------------------------------------------
+    beforeAll(async () => {
+        await page.goto(baseUrl);
+    });
+    //-------------------------------------------------------------------------
+    test("hello-notify on server --> history updated", async () => {
```

Comment:
These tests should run on the server-e2e tests (i.e. IIS, Docker) to verify the setup is correct.

Reply:
Done with 8410736166bca7dc9e57c48ff6698fc5d628c726